yolog/main.py

```diff
@@ -45,5 +46,7 @@ def main():
     git_arguments = sys.argv[1:]
     if git_arguments and (sys.argv[1] == "-h" or sys.argv[1] == "--help"):
         print(help_description)
+    elif git_arguments and (sys.argv[1] == "-config"):
+    	ConfigHandler()
```

Comment:
Similarly in this commit itself, bring the `yolog_gen` variable from above and declare it inside `else` block here ^. No variables should be in global namespace.